operator/roles/default/ossmconsole-deploy/templates/openshift/configmap-plugin.yaml

```
@@ -7,4 +7,5 @@ metadata:
 data:
   plugin-config.json: |
     {
+      "graph": "pf"
```

you want ossmconsole_vars.kiali.graph here.. something like:
```suggestion
      "graph": "{{ ossmconsole_vars.kiali.graph }}"
```

---

Done